packages/meditrak-app/app/database/types/OptionSet.js

```diff
@@ -24,7 +24,11 @@ OptionSet.schema = {
   properties: {
     id: 'string',
     name: { type: 'string', default: 'OptionSet not properly synced' },
-    options: 'Option[]',
+    options: {
```

Comment:
I presume the order the options are added doesn't matter, because they are ordered by `sortOrder` when pulled from the db?

Reply:
Sorry, I lost track of your comments in my email. Yep I think so, and when Autocomplete question is rendered, it always sorts the options by sortOrder